python/semantic_kernel/connectors/ai/embeddings/embedding_generator_base.py

```diff
@@ -12,6 +14,9 @@
     from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
 
 
+@deprecated(
+    "This class has been moved to semantic_kernel.connectors.ai.embedding_generator_base. Please update your imports."
+)
 @experimental
 class EmbeddingGeneratorBase(AIServiceClientBase, ABC):
```

Comment:
it doesn't make sense to keep the implementation here, you could do the import from the new spot and then call deprecated as a func around it:
```suggestion
EmbeddingGeneratorBase = deprecated(..., EmbeddingGeneratorBase)
```